src/main/java/connor/Connor.java

```diff
@@ -0,0 +1,113 @@
+package connor;
+
+import connor.command.ByeCommand;
+import connor.exception.InvalidTaskFileException;
+import connor.task.TaskList;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.util.Scanner;
+
+/**
+ * Main class for Connor program to start running.
+ *
+ * @author jaysmyname
+ * @version 1.0.0
+ */
+public class Connor {
+    public static final String CURRENT_VERSION = "Version 1.0.0";
+    public static final String LINE = "_".repeat(66);
+    public static final String INDENT = " ".repeat(4);
+    public static final String INPUT_HERE = ">>> ";
+
+    private static final String TASK_FILEPATH = "data/connor.txt";
+
+    private static final String ERROR_FILE_NOT_FOUND = "Error! Task file not found!";
+
+    private static boolean isActive = true;
+    private static final Scanner sc = new Scanner(System.in);
+
+    private Storage storage;
+    private Ui ui;

```

Comment:
Variables are appropriately named. All using camel case and booleans have the is prefix.